datadog-monitors-monitor-handler/CHANGELOG.md

```diff
@@ -1,10 +1,17 @@
 # Changelog
 
+# 1.0.2
+
+Released on 2019-11-21
+
+[Link to Resource] s3://datadog-cloudformation-resources/datadog-monitors-monitor/datadog-monitors-monitor-1.0.2.zip
+[BUGFIX] Use `Double` for all previously `Float` values to get sufficient precision for high thresholds.
```

Comment:
Is is just thresholds ?

Reply:
Yeah. Feel free to see the commit: https://github.com/DataDog/datadog-cloudformation-resources/commit/0e31ad85cfbd8cadac2e61f2f908113626623222

Reply:
Ahh ok, sorry. When I read I assumed Critical threshold, not all. LGTM